### PR TITLE
Bug 1888542: decrease CMO log verbosity from 3 to 2

### PR DESCRIPTION
**Pull request by simonpasquier**
Manual cherry-pick of #913. The automatic cherry-pick [didn't work](https://github.com/openshift/cluster-monitoring-operator/pull/913#issuecomment-708961885).

* [ ] I added CHANGELOG entry for this change.
* [X] No user facing changes, so no entry in CHANGELOG was needed.

### CODE REVIEW

**Review by paulfantom — APPROVED** [review]
/lgtm

**Review by lilic — COMMENTED** [review]
/lgtm

**Comment by juzhao** [comment]
@simon, tested with the PR, and launched cluster via `launch openshift/cluster-monitoring-operator#961 aws`, issue still happen
```
# oc -n openshift-monitoring get deploy cluster-monitoring-operator -oyaml | grep "logtostderr" -A3
--
        - -logtostderr=true
        - -v=2
```

```
# oc -n openshift-monitoring logs $(oc -n openshift-monitoring get po | grep cluster-monitoring-operator | awk '{print $1}') -c cluster-monitoring-operator | grep "Throttling request took" | tail
I1027 07:33:20.019624       1 request.go:621] Throttling request took 1.790137417s, request: PUT:https://172.30.0.1:443/apis/rbac.authorization.k8s.io/v1/clusterroles/prometheus-operator
I1027 07:33:21.019629       1 request.go:621] Throttling request took 1.997418933s, request: PUT:https://172.30.0.1:443/apis/rbac.authorization.k8s.io/v1/clusterroles/node-exporter
I1027 07:33:22.019646       1 request.go:621] Throttling request took 1.961985562s, request: GET:https://172.30.0.1:443/apis/rbac.authorization.k8s.io/v1/clusterrolebindings/prometheus-operator
I1027 07:33:23.219611       1 request.go:621] Throttling request took 1.797505153s, request: PUT:https://172.30.0.1:443/apis/rbac.authorization.k8s.io/v1/clusterroles/alertmanager-main
I1027 07:33:24.220006       1 request.go:621] Throttling request took 1.366936784s, request: GET:https://172.30.0.1:443/apis/rbac.authorization.k8s.io/v1/clusterroles/monitoring-rules-edit
I1027 07:33:25.419656       1 request.go:621] Throttling request took 1.391449253s, request: POST:https://172.30.0.1:443/apis/rbac.authorization.k8s.io/v1/clusterrolebindings
I1027 07:33:26.619653       1 request.go:621] Throttling request took 1.361435866s, request: GET:https://172.30.0.1:443/apis/rbac.authorization.k8s.io/v1/clusterrolebindings/telemeter-client-view
I1027 07:33:31.592527       1 request.go:621] Throttling request took 1.170534334s, request: GET:https://172.30.0.1:443/api/v1/namespaces/openshift-monitoring/serviceaccounts/prometheus-adapter
I1027 07:33:32.592538       1 request.go:621] Throttling request took 1.195860831s, request: DELETE:https://172.30.0.1:443/api/v1/namespaces/openshift-user-workload-monitoring/services/prometheus-user-workload
I1027 07:33:33.592598       1 request.go:621] Throttling request took 1.180102809s, request: GET:https://172.30.0.1:443/api/v1/namespaces/openshift-monitoring/configmaps/telemeter-trusted-ca-bundle
```

**Comment by simonpasquier** [comment]
Looking at the k8s client code, it will always log an info message whenever a request takes more than 1 second to complete. The interval and log level are hardcoded.

https://github.com/openshift/cluster-monitoring-operator/blob/f81ad9664b37edfc55d753d3e8db22864cbc4f18/vendor/k8s.io/client-go/rest/request.go#L57-L58

https://github.com/openshift/cluster-monitoring-operator/blob/f81ad9664b37edfc55d753d3e8db22864cbc4f18/vendor/k8s.io/client-go/rest/request.go#L556-L563

**Comment by juzhao** [comment]
thanks for the explanation, since request takes more than 1 second to complete will always logged, and there is less logs with the fix, add there are less logs compared with the OCP nightly build which does not have the fix, add /lgtm and wait for the PR merged to 4.5

**Pull request merged**
